DEVX-1942: Create first producer application (#491)
confluentinc/kafka-tutorials: _includes/tutorials/kafka-producer-application/kafka/markup/dev/create-topic.adoc

> In this step we're going to create a topic for use during this tutorial.

Reviewer: It reads strangely to say we're going to do something but then do something else.  So FYC, remove this line.  Then see comments below

> But first, you're going to open a shell on the broker docker container.

Reviewer: FYC, Delete this line

> Open a new terminal and window then run this command:

Reviewer: ```suggestion
Open a new terminal window and then run this command to open a shell on the broker docker container
```

+++++
<pre class="snippet"><code class="shell">{% include_raw tutorials/kafka-producer-application/kafka/code/tutorial-steps/dev/open-docker-shell.sh %}</code></pre>
+++++

> Now use the following command to create the topic for the producer to write to

Reviewer: ```suggestion
Next, create the topic that the producer to write to
```

+++++
<pre class="snippet"><code class="shell">{% include_raw tutorials/kafka-producer-application/kafka/code/tutorial-steps/dev/create-topic.sh %}</code></pre>
+++++

> Keep this terminal window open as you'll need to run a console conumer to verify your producer application in a few steps.

Reviewer: ```suggestion
Keep this terminal window open for later use when you run a console consumer to verify your producer application.
```
